sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -723,6 +725,11 @@ class SparkSession private(
    * @since 2.0.0
    */
   def stop(): Unit = {
+    if (!stopped.compareAndSet(false, true)) {
+      logInfo("SparkSession already stopped.")
+      return
```

Comment:
Hi, @arunmahadevan . Could you revise this to avoid `return` statement?

Reply:
Changed it.